app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue

```diff
@@ -157,16 +157,23 @@ export default {
       return this.metaTags.map(item => item.name);
     },
   },
+  watch: {
+    article: {
+      handler() {
+        if (!isEmptyObject(this.article.meta || {})) {
+          const {
+            meta: { title = '', description = '', tags = [] },
+          } = this.article;
+          this.metaTitle = title;
+          this.metaDescription = description;
+          this.metaTags = this.formattedTags({ tags });
+        }
+      },
+      deep: true,
+      immediate: true,
+    },
+  },
```

Comment:
@scmmishra Added a small suggestion (Feel free to ignore it if it's unnecessary).

```suggestion
  watch: {
    article: {
      handler() {
        const { meta = {} } = this.article;
        if (!isEmptyObject(meta)) {
          const { title = '', description = '', tags = [] } = meta;
          this.metaTitle = title;
          this.metaDescription = description;
          this.metaTags = this.formattedTags({ tags });
        }
      },
      deep: true,
      immediate: true,
    },
  },
```